relay/paseo/src/lib.rs

```diff
@@ -3191,7 +3210,7 @@ mod remote_tests {
 	async fn try_fast_unstake_all() {
 		sp_tracing::try_init_simple();
 		let transport: Transport =
-			var("WS").unwrap_or("wss://paseo.rpc.amforc.com:443".to_string()).into();
+			var("WS").unwrap_or("wss://rpc.paseo.io:443".to_string()).into();
```

Comment:
I don't believe this rpc is publicly accessible ?

Reply:
Fixed!